docs/Splashkit/Applications/npm-installation-guide.md

```diff
@@ -0,0 +1,153 @@
+# Npm installation guide: Windows and Mac
```

Comment:
**nitpick (non-blocking):** I think we can drop the Windows and Mac suffix. The breakdown of the section below would be good enough. Please feel free to ignore if you disagree.

```suggestion
# npm installation guide
```